.github/workflows/build-nightly.yml

```diff
@@ -28,6 +28,11 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Generate nightly version number
+        id: version_vars
+        run: |
+          echo mastodon_nightly_version=nightly-$(TZ="Etc/UTC" date +'%Y-%m-%d') >> $GITHUB_OUTPUT
```

Comment:
I think it would be cleaner to set `TZ` either in this step's `env` key, or globally for the workflow

Reply:
In theory we don't even need TZ set as it should default to UTC, but I guess we could set it in env too.